src/openforms/js/components/admin/form_design/FormStepsNav.js

```diff
@@ -57,7 +81,13 @@ const FormStepsNav = ({ steps=[], active=null, onActivateStep, onReorder, onDele
                     steps.map( (step, index) => (
                         <FormStepNavItem
                             key={index}
-                            name={ step.isNew ? `${step.name} [nieuw]` : step.name}
+                            name={step.isNew
```

Comment:
This currently causes a warning, because `FormStepNavItem` has `name: PropTypes.string.isRequired` in the prop types

Reply:
changed proptype to `node` instead of `string`